frontend/src/types.ts

```diff
@@ -1927,8 +1927,8 @@ export interface Experiment {
         recommended_sample_size?: number
         feature_flag_variants: MultivariateFlagVariant[]
     }
-    start_date?: string
-    end_date?: string
+    start_date?: string | null
```

Comment:
Q: why introduce `null` when `undefined` already is here?

Reply:
I think the similar comparison is like when we set `null` vs `blank` in django for models. So in the frontend for this if we set it to undefined we don't have to pass in this key value at all vs if we set it to null we have to pass in this key value even if it's a null setting. Right now when we create an experiment I think we pass in { ....otherExperimentData, start_date: start_date } with no end_date but otherwise we would have to set { ...otherExperimentData, start_date: start_date, end_date: null } because of typing. 

I guess technically we could still pass in `end_date: undefined` instead? But setting it to null is better practice